.github/workflows/reusable_build_packages.yaml

```diff
@@ -139,10 +238,10 @@ jobs:
       - name: Prepare project
         run: |
           cmake -B build -S . \
-                -DCMAKE_BUILD_TYPE=${{ inputs.build_type }} \
+                -DCMAKE_BUILD_TYPE=Release \
```

Comment:
Do we want to enforce jemalloc for musl build?

Reply:
Not sure but I'm not a jemalloc expert. What do you think?

Reply:
I think that all of our release artifact should spot the same behavior, therefore i'd say yes!